ArduCopter/Parameters.cpp

```diff
@@ -1215,6 +1215,15 @@ const AP_Param::GroupInfo ParametersG2::var_info2[] = {
     // @User: Advanced
     AP_GROUPINFO("TKOFF_THR_MAX", 6, ParametersG2, takeoff_throttle_max, 0.9),
 
+#if HAL_WITH_ESC_TELEM && FRAME_CONFIG != HELI_FRAME
+    // @Param: TKOFF_RPM_MAX
+    // @DisplayName: Takeoff Check RPM maximum
+    // @Description: Takeoff is not permitted until motors report at least this RPM.  Set to zero to disable check
```

Comment:
Andy's already pointed out the description has a typo but I think it's changed since then and is still incorrect.  Maybe "at least" should be "no more than"?

Reply:
Changed as suggested.